src/java/org/apache/cassandra/utils/FBUtilities.java

```diff
@@ -1405,15 +1406,21 @@ static Semver getKernelVersionFromUname()
         if (!isLinux)
             return null;
```

Comment:
it's kinda odd that we can return null here, but we fail parsing gcp cos_containerd's uname